src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -168,6 +176,22 @@ public function addContent($content, $type = null)
         }
     }
 
+    /**
+     * Force usage of the native HTML5 parser even if it wasn't detected.
+     */
+    public function enableHtml5Parser()
+    {
+        $this->useHtml5Parser = true;
+    }
+
+    /**
+     * Force usage of the native DOM parser even if the HTML5 parser is available.
+     */
+    public function disableHtml5Parser()
+    {
+        $this->useHtml5Parser = false;
```

Comment:
should we warn you if you call this method on a crawler which already has its content (and so it is too late to change the parsing) ?
FYI, the detection would be based on `$this->document !== null`.

Reply:
@tgalopin what about that ?

Reply:
That's a good idea but I'm not sure what type of warning should be raised (exception ?).

Reply:
I updated the PR to use a constructor argument instead, ensuring this won't happen.